front/app/utils/validate.ts

```diff
@@ -1,3 +1,8 @@
-export function isValidEmail(email) {
+export function isValidEmail(email: string) {
   return /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email);
 }
+
+export function isValidPhoneNumber(phoneNumber: string) {
+  // current back-end implementation, see phone_service.rb
```

Comment:
Ideally we would ensure these are synchronised but I can't come up with an easy way to do that off the top of my head.

Reply:
True, but this is unlikely to change.

Reply:
I'll add a note in the back-end as well to inform FE folks when validation changes.